ServerCore/Pages/Events/Standings.cshtml.cs

```diff
@@ -17,15 +21,29 @@ public class StandingsModel : EventSpecificPageModel
 
         public SortOrder? Sort { get; set; }
 
+        public string LocationDisplayName { get; set; }
+
+        public List<SelectListItem> TeamLocationFilter { 
+            get
+            {
+                List<SelectListItem> values = new List<SelectListItem>();
+                values.Add(new SelectListItem("All Teams", "all"));
+                values.Add(new SelectListItem("Local Teams", "local"));
+                values.Add(new SelectListItem("Remote Teams", "remote"));
+                return values;
+            } 
+        }
+
         private const SortOrder DefaultSort = SortOrder.RankAscending;
 
         public StandingsModel(PuzzleServerContext serverContext, UserManager<IdentityUser> userManager) : base(serverContext, userManager)
         {
         }
 
-        public async Task OnGetAsync(SortOrder? sort)
+        public async Task OnGetAsync(SortOrder? sort, TeamLocation locationFilter)
```

Comment:
Is this a required parameter since it is not nullable? Does that mean that links to this page wouldn't work?

Reply:
That is an excellent question that did not occur to me, let me check.